test_scripts/API/SetGlobalProperties/AutoCompleteList/ATF_SetGlobalProperties_N_AutoCompleteList_InvalidValueType.lua

```diff
@@ -0,0 +1,52 @@
+---Verification Criteria
+---SDL must respond with INVALID_DATA and "success":"false
+---"autoCompleteList" is  wrong type
+-----------------------------------------------------------------------------------------------
+---[[ General configuration parameters ]]
+config.deviceMAC = "12ca17b49af2289436f303e0166030a21e525d266e209267433801a8fd4071a0"
+
+--[[ Required Shared libraries ]]
+local commonFunctions = require ('user_modules/shared_testcases/commonFunctions')
+local commonSteps = require('user_modules/shared_testcases/commonSteps')
+
+--[[ General Precondition before ATF start ]]
+config.defaultProtocolVersion = 2
+commonSteps:DeleteLogsFileAndPolicyTable()
+
+--[[ General Settings for configuration ]]
+Test = require('connecttest')
+require('cardinalities')
+require('user_modules/AppTypes')
+
+--[[ Test ]]
+commonFunctions:newTestCasesGroup("Test")
+function Test:AutoCompleteList_IsInvalidType()
+--mobile side: sending SetGlobalProperties request
+local cid = self.mobileSession:SendRPC("SetGlobalProperties",
+    {
+      keyboardProperties =
+      {
+        keyboardLayout = "qwerty",
+        keypressMode = "single_keypress",
+        limitedCharacterList =
+        {
+          "a"
+        },
+        language = "EN-US",
+        autoCompleteText = "Text_1, Text_2",
+        autoCompleteList = 5
+      }
+    })
+--mobile side: expect SetGlobalProperties response
+   EXPECT_RESPONSE(cid, { success = false, resultCode = "INVALID_DATA" })
+   --mobile side: expecting OnHashChange notification
```

Comment:
@Aleksandriys, no need to specify comment about waiting OnHashChange if in result you don't expect it: Times(0)